{{ cookiecutter.formal_name }}/Dockerfile

```diff
@@ -11,43 +11,45 @@ ARG PY_VERSION
 ARG SYSTEM_REQUIRES
 
 # Make sure installation of tzdata is non-interactive
-ENV DEBIAN_FRONTEND="noninteractive"
+ENV DEBIAN_FRONTEND="noninteractive" \
+# Disable pip warnings about running as root (pip >= 22.1)
+    PIP_ROOT_USER_ACTION="ignore" \
+# Disable pip progress bar
+    PIP_PROGRESS_BAR="off"
 
 # Install the deadsnakes PPA so we can get arbitrary Python versions
 RUN apt-get update -y && \
-    apt-get install -y \
+    apt-get install --no-install-recommends -y \
         software-properties-common \
         dirmngr \
         apt-transport-https \
         lsb-release \
-        ca-certificates
-RUN apt-add-repository ppa:deadsnakes/ppa
-
-# Install git, Python, and any packages required
-RUN apt-get update -y && \
-    apt-get install -y \
+        ca-certificates && \
+    apt-add-repository ppa:deadsnakes/ppa && \
+# Install git, file (for linuxdeploy), Python, and any packages required
+    apt-get update -y && \
+    apt-get install --no-install-recommends -y \
         git \
+        file \
         python${PY_VERSION}-dev \
         python${PY_VERSION}-venv \
         ${SYSTEM_REQUIRES}
 
 # Install (and update) pip, disabling the global cache
-RUN python${PY_VERSION} -m ensurepip
-RUN python${PY_VERSION} -m pip config set global.cache-dir false
-RUN python${PY_VERSION} -m pip install --upgrade pip
-RUN python${PY_VERSION} -m pip install --upgrade setuptools
-RUN python${PY_VERSION} -m pip install --upgrade wheel
+RUN python${PY_VERSION} -m ensurepip && \
+    python${PY_VERSION} -m pip config set global.cache-dir false && \
+    python${PY_VERSION} -m pip install --upgrade pip && \
+    python${PY_VERSION} -m pip install --upgrade setuptools wheel
 
 # Ensure Docker user UID:GID matches host user UID:GID (beeware/briefcase#403)
 # Use --non-unique to avoid problems when the UID:GID of the host user
 # collides with entries provided by the Docker container.
 ARG HOST_UID
 ARG HOST_GID
-RUN groupadd --non-unique --gid $HOST_GID briefcase
-RUN useradd --non-unique --uid $HOST_UID --gid $HOST_GID brutus --home /home/brutus
-
+RUN groupadd --non-unique --gid $HOST_GID briefcase && \
+    useradd --non-unique --uid $HOST_UID --gid $HOST_GID brutus --home /home/brutus && \
 # Create the brutus home directory; ensure that it is owned by brutus.
-RUN mkdir -p /home/brutus && chown brutus:briefcase /home/brutus
+    mkdir -p /home/brutus && chown brutus:briefcase /home/brutus
```

Comment:
Comments can be indented inside RUN commands as shown [here](https://stackoverflow.com/a/53564789). But in this case the comment adds nothing, so it might as well be removed.

Reply:
True; removed comment.